wallet-new/src/Cardano/Wallet/API/V1/Types.hs

```diff
@@ -1956,14 +2379,15 @@ instance Arbitrary Redemption where
 --
 
 type family Update (original :: *) :: * where
-  Update Wallet        = WalletUpdate
-  Update Account       = AccountUpdate
-  Update WalletAddress = () -- read-only
+    Update Wallet        = WalletUpdate
+    Update Account       = AccountUpdate
+    Update WalletAddress = () -- read-only
 
 type family New (original :: *) :: * where
-  New Wallet  = NewWallet
-  New Account = NewAccount
-  New WalletAddress = NewAddress
+    New Wallet         = NewWallet
+    New Account        = NewAccount
+    New WalletAddress  = NewAddress
+    New ExternalWallet = NewExternalWallet
```

Comment:
now is a good time to put the result on a newline so we can avoid reindenting every line when we add a new case, eg:

```haskell
type family New a where
    New Wallet =
        NewWallet
    New Account =
        NewAccount
    New ExternalWallet =
        NewExternalWallet
```

Reply:
Fixed.